### PR TITLE
kubectl-minio command to register tenants in SUBNET

### PR DESCRIPTION
**Pull request by reivaj05**
Adds new kubectl-minio to register tenants in SUBNET

kubectl minio license register (It will ask for SUBNET credentials to retrieve API KEY)
kubectl minio license register --api-key {API_KEY}

Closes: https://github.com/miniohq/engineering/issues/917

### CODE REVIEW

**Review by dvaldivia — DISMISSED** [review]

**Review by Alevsk — DISMISSED** [review]
Please address the following code issues

```
Running lint check
Building 'kubectl-minio' binary
?       github.com/minio/kubectl-minio  [no test files]
?       github.com/minio/kubectl-minio/cmd      [no test files]
?       github.com/minio/kubectl-minio/cmd/helpers      [no test files]
?       github.com/minio/kubectl-minio/cmd/resources    [no test files]
cmd/helpers/helpers.go:253:1: exported: comment on exported function AskNumber should be of the form "AskNumber ..." (revive)
// Ask user for number input."
^
cmd/helpers/helpers.go:296:1: exported: exported function GetTenantConfiguration should have comment or be unexported (revive)
func GetTenantConfiguration(ctx context.Context, tenant *miniov2.Tenant, clientSet *kubernetes.Clientset) (map[string][]byte, error) {
^
cmd/helpers/helpers.go:264:11: indent-error-flow: if block ends with a return statement, so drop this else and outdent its block (revive)
                        } else {
                                return nil
                        }
cmd/helpers/helpers.go:284:11: indent-error-flow: if block ends with a return statement, so drop this else and outdent its block (revive)
                        } else {
                                return nil
                        }
make: *** [Makefile:76: plugin] Error 1
```

you can run the `make` command in the operator folder to check theres no more issues once the above issues are fixed, Im adding a test to run this check automatically

**Review by pjuarezd — CHANGES_REQUESTED** [review]

File: kubectl-minio/cmd/license-register.go [hunk 1]
```diff
@@ -0,0 +1,189 @@
...
+	}
+	return d.registerTenants(tenants.Items, kubeClient)
+}
+
+func (d *licenseRegisterCmd) registerTenants(tenants []v2.Tenant, kubeClient *kubernetes.Clientset) error {
+	ctx := context.Background()
+	for _, tenant := range tenants {
+		fmt.Printf("Registering tenant %s\n", tenant.Name)
+		if err := d.registerTenant(ctx, tenant, kubeClient); err != nil {
+			return err
```
[the remark this answers is not in the document]
Comment (reivaj05) [reply kubectl-minio/cmd/license-register.go:109]: Should be fine  to just continue to the next tenant on error for now

File: kubectl-minio/cmd/license-register.go [hunk 2]
```diff
@@ -0,0 +1,189 @@
...
+		}
+		fmt.Printf("Tenant %s registered\n", tenant.Name)
+	}
+	return nil
+}
+
+func (d *licenseRegisterCmd) registerTenant(ctx context.Context, tenant v2.Tenant, kubeClient *kubernetes.Clientset) error {
+	adminClient, err := d.getTenantAdminClient(ctx, tenant, kubeClient)
```
[the remark this answers is not in the document]
Comment (reivaj05) [reply kubectl-minio/cmd/license-register.go:117]: Your minio service is not reachable locally, I use [kubefwd](https://kubefwd.com/) to expose services locally

**Comment by cniackz** [comment]
@reivaj05 please resolve the conflicts 👍

**Pull request closed without merging**